Userland/Utilities/diff.cpp

```diff
@@ -37,11 +37,29 @@ int main(int argc, char** argv)
 
     auto hunks = Diff::from_text(file1->read_all(), file2->read_all());
     for (const auto& hunk : hunks) {
```

Comment:
Nitpick about the commit message: I personally prefer just `diff: Blah blah` over `Utilities/diff: Blah blah`, but I guess this is fine. Feel free to click "resolve" without changing it, if you prefer it the current way.